x-pack/plugins/apm/public/components/routing/home/index.tsx

```diff
@@ -138,8 +138,8 @@ export const home = {
           refreshPaused: t.union([t.literal('true'), t.literal('false')]),
           refreshInterval: t.string,
           comparisonEnabled: toBooleanRt,
-          comparisonType: comparisonTypeRt,
         }),
+        offsetRt,
```

Comment:
I can try adding some redirects for all these

Reply:
do you mean for comparisonType? If it's easy go for it.

Reply:
Done in https://github.com/elastic/kibana/pull/127812/commits/6adb3148bc95b15996f607c0aaf55c80c33d7e4c